about.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./about.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>Capstone</title>
+</head>
+<body>
+
+    <div class="containerr">
+
+    
+
+    <nav class="navbar">
+
+        <button class="hamburger">
+            <span><i class="fa fa-bars fa-2x"></i></span>
+          </button>   
+
+          <button class="close">
+            &times;
+          </button>
+
+        
+          <ul class="social-media-contact">
+            <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+
+
+            <div class="nav-desktop-container" >
+
+                <div class="hide-mobile"><a href="#" ><img id="navbar-logo" src="img/hubblespace.png" alt="hubble"></a></div>
+                <div>
+                    <ul class="desktop-nav ">
+                    
+                        <li><a class="mnitem about-bar" href="index.html">Home</a></li>
+                        <li><a class="mnitem" href="index.html#mainprogram">Program</a></li>
+                        <li><a class="mnitem" href="index.html#mainprogram">Accomodation</a></li>
+                        <li><a class="mnitem" href="index.html#fund">Funding</a></li>
+                        <li><a class="mnitem register-bar" href="#">Register</a></li>
+                        
+
+                    </ul>
+                </div>
+            </div>
+
+    </nav>
```

Comment:
- [ ] On the desktop the navbar links are not all fitting on the page check out this image and make necessary changes
![spa](https://user-images.githubusercontent.com/44745855/132730517-b1faebf0-35cd-41c3-9705-e45a55039194.png)